02/app.js

```diff
@@ -1,7 +1,19 @@
 const randomArray = createRandomArray();
+
 console.log(randomArray);
 
 
+for(i=0; i<randomArray.length; i++) {
```

Comment:
brakuje przed `i` słowa `let` tj. `for(let i=0)` - to bardzo ważne bo inaczej tworzymy zmienną globalną